File: 110A_Nearly_Lucky_Number/solution.c
#include <stdio.h>
int main() {
    char s[101];
    scanf("%s", &s);
    int n = 0;
    for (int i = 0; i < strlen(s); i++) {
        if (s[i] == '4' || s[i] == '7') n++;
        if (n > 7) break;
    }
    printf("%s", (n == 4 || n == 7) ? "YES" : "NO");
    return 0;
}

File: 110A_Nearly_Lucky_Number/solution.py
s = input()
n = 0
for i in range(len(s)):
    if s[i] == '4' or s[i] == '7':
        n += 1
    if n > 7: break
if n == 4 or n == 7:
    print("YES")
else:
    print("NO")